.vscode/launch.json

```diff
@@ -23,8 +23,8 @@
                 "<node_internals>/**"
             ],
             "env": {
-                // Enable this to turn on redux logging during debugging
-                "XVSC_JUPYTER_FORCE_LOGGING": "1",
+                // Disable this to turoff on redux & console logging during debugging
+                "VSC_JUPYTER_FORCE_LOGGING": "1",

```

Comment:
I use the console log a lot, as it helps me see errors & warnings (colors in console window).
I don't see why turning this on is a problem for debugging, hence leaving as deafult. else I end up turning this on always & not checking this in..